content/docs/projects/trust.md

```diff
@@ -48,14 +48,27 @@ spec:
       -----END CERTIFICATE-----
   target:
     # Data synced to the ConfigMap `my-org.com` at the key `root-certs.pem` in
-    # every namespace.
+    # every namespace that has the label "linkerd.io/inject=enabled".
```

Comment:
nitpick: it might be worth adding this to `content/next-docs/projects/trust.md` as well, but that only matters if this PR goes in before 1.9 and even then it'll be pretty simple for the releaser to fix!